src/main/java/net/uaznia/lukanus/hudson/plugins/gitparameter/GitParameterDefinition.java

```diff
@@ -290,10 +300,12 @@ public String prettyRevisionInfo(GitClient newgit, Revision r) {
 					return Collections.singletonMap(errMsg, errMsg);
 				}
 
-				FetchCommand fetch = newgit.fetch_().from(remoteURL,
-						repository.getFetchRefSpecs());
-				fetch.execute();
 				if (type.equalsIgnoreCase(PARAMETER_TYPE_REVISION)) {
+					// If we use the GitClient#getRemoteReferences function, we don't
```

Comment:
71d933a840c263e0f3f3ce6998a42a7070a19648 log says "_Add an optional "prefer remote" flag to prefer using ls-remote if possible_" - I can't see any reason this would not be possible - ls-remote is a standard git command - can you explain ?